zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -0,0 +1,30 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Initial Function implmention. Will be evolved and documented more in depth"""
+from qiskit.circuit.library import Barrier
+from qiskit.dagcircuit import DAGCircuit
+
+
+def apply_folding(noisy_dag: DAGCircuit, original_dag: DAGCircuit, copies: int = 2) -> DAGCircuit:
+    """Initial function for now, will be evolved and documented more in depth"""
+
```

Comment:
Try to avoid in-function whitespace! Functions should be brief enough so that they do not need it 🏳️